static/scripts/csrf.js

```diff
@@ -0,0 +1,18 @@
+function inIframe() {
+	try {
+		return window.self !== window.top;
+	} catch (e) {
+		return true;
+	}
+}
+$('document').ready(() => {
+	const retrybtn = $('#retrybtn');
+	retrybtn.html('Erneut versuchen');
```

Comment:
What if retry fails and fails again? Can u create a link back to the last page with prefill the form with given values? we also would not need a separate csrf error page if the current form will be shown again.

Reply:
Don't think that will work. Would require changes in all forms on the page since the server would have to provide the pre-filled inputs. 
When a page is actually reloaded (required to get a new csrf token), all form data is lost.

Reply:
Added detection of again failed form. Then displays a standard reload button.